src/dodal/devices/i19/shutter.py

```diff
@@ -1,61 +1,86 @@
+    This device will send a REST call to the blueapi instance controlling the optics \
+    hutch running on the I19 cluster, which will evaluate the current hutch in use vs \
+    the hutch sending the request and decide if the plan will be run or not.
     As the two hutches are located in series, checking the hutch in use is necessary to \
     avoid accidentally operating the shutter from one hutch while the other has beamtime.
 
-    The hutch name should be passed to the device upon instantiation. If this does not \
-    coincide with the current hutch in use, a warning will be logged and the shutter \
-    will not be operated. This is to allow for testing of plans.
-    An error will instead be raised if the hutch state reads as "INVALID".
+    The name of the hutch that wants to operate the shutter should be passed to the \
+    device upon instantiation.
+
+    For details see the architecture described in \
+    https://github.com/DiamondLightSource/i19-bluesky/issues/30.
     """
 
     def __init__(self, prefix: str, hutch: HutchState, name: str = "") -> None:
-        self.shutter = HutchShutter(prefix=prefix, name=name)
-        bl_prefix = prefix.split("-")[0]
-        self.hutch_state = epics_signal_r(str, f"{bl_prefix}-OP-STAT-01:EHStatus.VALA")
-        if hutch == HutchState.INVALID:
-            raise HutchInvalidError(
-                "Cannot define experimental shutter for invalid hutch"
-            )
+        self.shutter_status = epics_signal_r(ShutterState, f"{prefix}STA")
         self.hutch_request = hutch
+        self.url = OPTICS_BLUEAPI_URL
         super().__init__(name)
 
+    async def read(self) -> dict[str, Reading]:
+        default_reading = await super().read()
+        status = await self.shutter_status.get_value()
+        return {
+            "shutter_status": Reading(value=status, timestamp=time.time()),
+            **default_reading,
+        }
+
     @AsyncStatus.wrap
     async def set(self, value: ShutterDemand):
-        hutch_in_use = await self.hutch_state.get_value()
-        LOGGER.info(f"Current hutch in use: {hutch_in_use}")
-        if hutch_in_use == HutchState.INVALID:
-            raise HutchInvalidError(
-                "The hutch state is invalid. Contact the beamline staff."
-            )
-        if hutch_in_use != self.hutch_request:
-            # NOTE Warn but don't fail
-            LOGGER.warning(
-                f"{self.hutch_request} is not the hutch in use. Shutter will not be operated."
-            )
-        else:
-            await self.shutter.set(value)
+        REQUEST_PARAMS = {
+            "name": "operate_shutter_plan",
+            "params": {
+                "experiment_hutch": self.hutch_request.value,
+                "access_device": "access_control",  # Device name in i19-blueapi
+                "shutter_demand": value,
+            },
+        }
+        async with ClientSession(base_url=self.url, raise_for_status=True) as session:
```

Comment:
Could: We're going to need this logic for triggering a plan from a device a lot. I suspect very soon we will end up with a  common `BlueAPIBackedDevice` that this inherits from. Could do that here or in the next PR for this architecture.

Reply:
I think I'd rather do this in a separate ticket - issue incoming